__tests__/fixture/lends.input.js

```diff
@@ -6,6 +6,9 @@
 export default TheClass(
   /** @lends TheClass.prototype */
   {
+    /** My field */
+    // prettier-ignore
```

Comment:
without prettier-ignore `"myField"` will be changed into `myField`

Reply:
how about naming the field "my field"? Prettier only unquotes properties that are valid identifiers.

Reply:
right, thanks!